content/glossary.md

```diff
@@ -8,12 +8,14 @@ date = "2018-05-14T16:11:08-05:00"
 +++
 
 **Tracing**  
-Tracking the entire path of execution of a service or a function from start until end. e.g. During an upload, tracking when a file handle is received, opened, sanitized and uploaded to the cloud.  
+
+Distributed Tracing tracks the progression of a request as it is handled by the services and processes that make up a distributed system. Each step in the Trace is called a Span. Each Span contains metadata, such as the length of time spent in the step, and optionally annotations about the work being performed. This information can be used for debugging and performance tuning of distributed systems.
 
 ---
 
 **Span**  
-The unit of traversal during tracing. Each span might have a children spans within it or it might be the first in its generation and called a root.  
+
+A unit work in a trace. Each span might have a children spans within it or it might be the first in its generation and called a root span.  
```

Comment:
s/a children spans/children spans

Reply:
Also, this sentence is still confusing because root span can have children spans too. Maybe rephrase as "Each span might have children spans within it. If a span does not have a parent, it is the first in its generation and called a root span."?

Reply:
Done.